egs/chime6/asr1/conf/preprocess.yaml

```diff
@@ -0,0 +1,21 @@
+process:
+  # these three processes are a.k.a. SpecAugument
+  - type: "time_warp"
+    max_time_warp: 5
+    inplace: true
+    mode: "PIL"
+  - type: "freq_mask"
+    F: 30
+    n_mask: 2
+    inplace: true
+    replace_with_zero: false
+  - type: "time_mask"
+    T: 40
+    n_mask: 2
+    inplace: true
+    replace_with_zero: false
+  - type: "token_augment"
+    gtm_path: "/mnt/k05/laptev/exp/espnet_chime6_asr_recipe_exp/jsondata/train_worn_simu_u400k_cleaned_trim_sp/gtm.csv"
```

Comment:
This should be removed.
Probably this "token_augment" is an experimental function?

Reply:
Thanks for pointing out! It leaked from another experiment. We removed it.